setup.cfg

```diff
@@ -0,0 +1,86 @@
+[metadata]
+name = micro-sam
+version = attr: micro_sam.__version__
+description = SegmentAnything For Microscopy
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/computational-cell-analytics/micro-sam
+author = Anwai Archit, Constantin Pape
+author_email = yourname@example.com
+license = MIT
+license_files = LICENSE
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Framework :: napari
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Topic :: Scientific/Engineering :: Image Processing
+project_urls =
+    Bug Tracker = https://github.com/computational-cell-analytics/micro-sam/issues
+    Documentation = https://computational-cell-analytics.github.io/micro-sam/micro_sam.html
+    Source Code = https://github.com/computational-cell-analytics/micro-sam
+    User Support = https://github.com/computational-cell-analytics/micro-sam/issues
+
+[options]
+packages = find:
+install_requires =
```

Comment:
Many of these packages (python-elf, vigra, etc.) appear to be available on conda-forge but not PyPI.
I've commented the install requirement out right now, because I am doing a lot of `pip install -e .` while I test things. Ideally we should fix this part up properly.

Currently there are no requirements for these packages to be installed before micro-sam, you just run into import errors if they are not there.

Reply:
Indeed, some of my "stack" is not on pip. The underlying reasons are vigra and nifty:
- https://github.com/constantinpape/vigra
- https://github.com/DerThorsten/nifty

which are fairly complex C++ packages and hence not so easy to provide via pip.

What do you think is the best strategy here?

Reply:
I think the best strategy is to:
1. Distribute micro-sam to users via conda-forge
2. For developers, we put a small section in the docs somewhere that says how to make a development environment, and that will include what dependencies you need to install.